skll/learner.py

```diff
@@ -107,8 +109,14 @@
                         [{'C': [0.01, 0.1, 1.0, 10.0, 100.0]}],
                         SVR:
                         [{'C': [0.01, 0.1, 1.0, 10.0, 100.0],
-                          'gamma': [0.01, 0.1, 1.0, 10.0, 100.0]}]}
-
+                          'gamma': [0.01, 0.1, 1.0, 10.0, 100.0]}],
+                        RidgeCV:
+                        [{'alphas': [(0.01, 0.1, 1.0, 10.0, 100.0)]}],
+                        BayesianRidge:
+                        [{'alpha_1': [1.e-4, 1.e-5, 1.e-6, 1.e-7, 1.e-8],
+                          'alpha_2': [1.e-4, 1.e-5, 1.e-6, 1.e-7, 1.e-8],
+                          'lambda_1': [1.e-4, 1.e-5, 1.e-6, 1.e-7, 1.e-8],
+                          'lambda_2': [1.e-4, 1.e-5, 1.e-6, 1.e-7, 1.e-8]}]}
 
```

Comment:
How did you come up with these values for these regressors? Are they just guesses or are they the sorts of values people try? You might have to read some papers to make sure.